docs/_static/css/custom.css

```diff
@@ -0,0 +1,6 @@
+@import url("default.css");
+
+/*Make sidebar captions steelblue*/
+span.caption-text {
+  color: steelblue;
```

Comment:
Do you have a screenshot on how this looks?

Reply:
![2019-03-15-121049_654x1044_scrot](https://user-images.githubusercontent.com/1026799/54427660-683c9280-471b-11e9-97eb-2c59ebe1e421.png)
Don't mind the weird line, that's a problem with my screenshot program.

Reply:
Looks nice!